optuna/progress_bar.py

```diff
@@ -75,14 +82,20 @@ def _init_valid(self) -> None:
         optuna_logging.disable_default_handler()
         optuna_logging._get_library_root_logger().addHandler(_tqdm_handler)
 
-    def update(self, elapsed_seconds: float) -> None:
+    def update(self, elapsed_seconds: float, study: "Study") -> None:
         """Update the progress bars if ``is_valid`` is :obj:`True`.
 
         Args:
             elapsed_seconds:
                 The time past since :func:`~optuna.study.Study.optimize` started.
+            study:
+                The current study object
         """
 
+        if not study._is_multi_objective():
+            msg = f"Best trial: {study.best_trial.number}. Best value: {study.best_value}"
+            self._progress_bar.set_description(msg)
+
```

Comment:
Could you move these lines to L100, specifically, just after `if self._is_valid:`? This will fix the failed CIs.

Reply:
Fixed

Reply:
Thanks!